app/model/Application.scala

```diff
@@ -27,6 +27,10 @@ import play.api.libs.json._
 import uk.gov.hmrc.play.json.Union
 import uk.gov.hmrc.time.DateTimeUtils
 
+// TODO: Is the default format for Joda correct for Gatekeeper?
+import play.api.libs.json.JodaReads._
+import play.api.libs.json.JodaWrites._
+
```

Comment:
Anybody got views on this?
Not sure just using Joda implicits will be correct in this context.